### ISSUE TITLE
Improve link styles (a11y)

### ISSUE THREAD

**Issue by aviannamiller** [omeka/theme-seasons#51]
Underlines (or some other type of non-color dependent distinguishing style) should be added for inline links.

Link to body text contrast should be at least 3:1, currently:
- Spring: 1.8:1, 1.7:1 on hover/focus
- Summer: 1:1
- Autumn: 1.6:1
- Winter: 1.1:1, 1.5:1 on hover/focus
- Night: 1.5:1, also an issue on hover/focus
